src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineLinuxLauncher.java

```diff
@@ -105,14 +96,15 @@ private boolean bootstrap(GoogleKeyPair kp, ComputeEngineComputer computer, Task
       }
       if (!isAuthenticated) {
         logWarning(computer, listener, "Authentication failed");
-        return false;
+        return Optional.empty();
       }
-    } finally {
+    } catch (Exception e) {
```

Comment:
In general I'm weary of swallowing exceptions like this. I think at the very least we should consider logging it as a warning.

Reply:
Added logging